package.json

```diff
@@ -17,5 +17,8 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "react-router-dom": "^6.3.0"
```

Comment:
root package.json should not be modified: restore it (as well as root package-lock.json)